scalatest/src/main/scala/org/scalatest/tools/RunnerJFrame.scala

```diff
@@ -319,7 +319,7 @@ private[scalatest] class RunnerJFrame(
               val (grayStackTraceElements, blackStackTraceElements) =
                 holder.throwable match {
                   case Some(throwable) =>
-                    val stackTraceElements = throwable.getStackTrace.toList
+                    val stackTraceElements = throwable.getStackTrace.map(_.nn).toList
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.